JenkinsService/src/main/java/com/revature/JenkinsServlet.java

```diff
@@ -0,0 +1,132 @@
+        log.info("Jenkins servlet started.");
+        JSONObject responseJson = new JSONObject();
+        parseParams(request);
+
+        try {
+            makeContainerRepo(responseJson);
+            makeJob(responseJson);
+            log.info("makeJob method and job name added to JSON successfully.");
+        } catch (Exception e) {
+            responseJson.put("errorMsg", e.getMessage());
+            log.error(e.getMessage());
+        }
+		
+        response.setContentType("application/json");
+        response.getWriter().write(responseJson.toString(0));
+        log.info("Jenkins servlet has written HTTP response and finished.");
+    }
+
+    private void parseParams(HttpServletRequest req) throws IOException {
+        log.info("Reading JSON from HTTP request.");
+        JSONObject json = parseRequestToJSON(req);
+        parseJsonToVars(json);
+    }
+
+    private JSONObject parseRequestToJSON(HttpServletRequest req) throws IOException {
+        StringBuffer jb = new StringBuffer();
+        String line = null;
+        try {
+            BufferedReader reader = req.getReader();
+            while ((line = reader.readLine()) != null)
+                jb.append(line);
+        } catch (Exception e) {
+            log.error("Exception in parseRequestToJSON method. Issue reading the HTTP request.  " + e.getMessage());
+            throw new IOException("Error reading input!");
+        }
+
+        JSONObject json = null;
+        try {
+            json = new JSONObject(jb.toString());
+        } catch (JSONException e) {
+            log.error("Request string is not JSON: " + jb.toString() + " " + e.getMessage());
+            String err = "Request string is not JSON: " + jb.toString();
+            throw new IOException(err);
+        }
+        return json;
+    }
+
+    private void parseJsonToVars(JSONObject json) throws IOException {
+        try {
+            repoUrl = json.getString("githubURL");
+            projName = json.getString("projectName").toLowerCase();
+            slackChannel = json.getString("slackChannel");
+            jenkinsUrl = json.getString("jenkinsURL");
+        } catch (JSONException e) {
+            log.error("Exception when trying to parse JSON to Java.  " + e.getMessage());
+            String err = "Error parsing JSON request string";
+            if (json != null) {
+                err += ": " + json.toString();
+            }
+            log.error("JSON exception occurring with this JSON: " + err);
+            throw new IOException(err);
+        }
+    }
+	
+    private void makeJob(JSONObject response) throws IOException {
+        log.info("Beginning makeJob method");
+        CommandExecutor exec = new CommandExecutor();
+        String cmd = "curl -X POST -u " + jenkinsAuth + " " + jenkinsUrl + "/job/seed/buildWithParameters --data githubURL=" + repoUrl + " --data projectName=" + projName + " --data slackChannel=" + slackChannel;
```

Comment:
This isn't wrong, but why are we using curl and not writing java code to do this?